corehq/apps/hqwebapp/templatetags/menu_tags.py

```diff
@@ -159,3 +160,14 @@ def aliased_language_name(lang_code):
             if code == lang_code:
                 return name
         raise KeyError('Unknown language code %s' % lang_code)
+
+
+@register.simple_tag(takes_context=True)
+def prelogin_url(context, urlname):
+    """
+    A prefix aware url tag replacement for prelogin URLs
+    """
+    if context.get('url_uses_prefix', False):
+        return reverse(urlname, args=[context['LANGUAGE_CODE']])
```

Comment:
It comes from [Django i18n context processor](https://docs.djangoproject.com/en/1.9/ref/templates/api/#django-template-context-processors-i18n)

Unless a view doesn't use (all prelogin view use) the `RequestContext` `LANGUAGE_CODE` should always be available. Will add a check and default to non-prefix URL